galaxy_api/api/pagination.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework import pagination
+from rest_framework.response import Response
+from rest_framework.utils.urls import remove_query_param, replace_query_param
+
+
+class LimitOffsetPagination(pagination.LimitOffsetPagination):
+    default_limit = 10
+    max_limit = 100
+
+    def get_first_link(self):
+        url = self.request.get_full_path()
```

Comment:
Looks like we're no longer returning the absolute path, at least according to https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.get_full_path. Is that correct? If so, +1

Reply:
http://platform-docs.cloud.paas.psi.redhat.com/rest/pagination.html shows "/api/myapp/v1/object/?limit=10&offset=0" style links, so seems right.

Reply:
We are returning the absolute **path**, but not the absolute URL (including scheme and netloc)